Makefile

```diff
@@ -293,7 +293,7 @@ prepare-and-launch-container-to-run-tests-within: container-test-build launch-co
 
 # all additional checks not called by prove
 .PHONY: test-checkstyle-standalone
-test-checkstyle-standalone: test-shellcheck test-yaml test-critic
+test-checkstyle-standalone: test-shellcheck test-yaml test-critic test-shfmt
```

Comment:
Please remove this. This has to be done in a followup PR. The container has to be rebuilt after this PR has been merged. Then it will be available in the following PR.